### ISSUE TITLE
Windows 10 64b + Skype

### ISSUE THREAD

**Issue by lefting** [alexazz19/SimpleSwitcher#10]
Skype виснет напрочь при переключении раскладки.

**Comment by alexazz19** [comment]
На win7 64 у меня работает нормально. Нужно будет посмотреть в чем там дело.

**Comment by alexazz19** [comment]
На счет исключений мысль хорошая и делать не сложно. Возможно, в каких либо играх тоже может оказаться полезным. Нужно будет реализовать.

**Comment by lefting** [comment]
На 7ке и 8ке то-же вроде не было проблем, перешел на 10ку и появились. Причем на 2х компьютерах. Виснет просто при попытке переключения языка по правму-левому ctrl.

**Comment by alexazz19** [comment]
Да, действительно стабильно виснет. Skype тупо зависает после того как ему отсылается сообщение с просьбой переключить язык:
PostMessage(m_hwndTop, WM_INPUTLANGCHANGEREQUEST, 0, (LPARAM)sendData.lay);

Но я попробовал, если переключать язык, эмулируя нажатие Alt+Shift, то все работает нормально.
Позже добавлю опцию, чтобы можно было так переключать язык, а пока можете добавить skype в исключения: опция disableInProgram (появилась в версии 1.0.5.0)

**Comment by lefting** [comment]
Спасибо за disableInProgram
